readme.md

```diff
@@ -115,7 +172,7 @@ Write some basic CSS that demonstrates changing a CSS property when the device w
 How is rebase different than a merge?
 
 Your Answer:
-```text
+```rebase is when you switch out the master for a new master branch
```

Comment:
Not quite. Rebasing involves taking one branch and re-building it's history starting from later commits. See the solution branch for a more detailed answer.